.mergify.yml

```
@@ -0,0 +1,13 @@
+pull_request_rules:
+  - name: Automatic merge on approval
+    conditions:
+      - base=master
+      - "#approved-reviews-by>=1"
+      - label=ready-to-merge
+      - label!=hold-off-merging
+      - status-success=CI / build
+    actions:
+      merge:
+        method: squash
```

I think we probably want to use the `rebase` method, which should automatically fall back to merge if rebase fails.
```suggestion
        method: rebase
```